src/accelerometer.rs

```diff
@@ -29,19 +21,19 @@ impl Accelerometer {
     pub fn new(
         gpioa: gpioa::Parts,
         gpioe: gpioe::Parts,
-        spi1: stm32::SPI1,
+        spi1: pac::SPI1,
         clocks: rcc::Clocks,
     ) -> Self {
-        let sck = gpioa.pa5.into_alternate().internal_pull_up(false);
-        let miso = gpioa.pa6.into_alternate().internal_pull_up(false);
-        let mosi = gpioa.pa7.into_alternate().internal_pull_up(false);
+        let sck = gpioa.pa5.internal_pull_up(false);
+        let miso = gpioa.pa6.internal_pull_up(false);
+        let mosi = gpioa.pa7.internal_pull_up(false);
```

Comment:
It wasn't clear to me whether we need these:
```rust
`internal_pull_up(false);
```

If nothing was using these pins before, the internal pull-ups should already be disabled.

I suppose, however, that this accelerometer module could be taking over the pins after they had been used for something else? Open to suggestions.